docs/scos/dev/front-end-development/202212.0/oryx/getting-started/oryx-environment.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Environment Variables"
+---
+
+Environment variables are useful when:
+
+- Values are different across developer machines.
+- Values are different across multiple environments: (dev, staging, qa, prod)
+- Values change frequently and are highly dynamic.
+
+Environment variables can be changed easily - especially when running in CI.
+
+Oryx uses [dotenv](https://github.com/motdotla/dotenv) for environment variables management. 
+
+## Storefront Variables 
+
+The list of environment variables to configure Storefront:
+
+| Variable | Required | Default | Description                               |
+|----------|----------|---------|-------------------------------------------|
+| `SCOS_BASE_URL` | Yes | None | Spryker GLUE API URL |
+| `STORE` | Yes | None | Default store name, e.g. DE, AT etc. The value is case-sensitive |
```

Comment:
Not required. We don't use it in boilerplate, as we fallback to default store in the backend.

Reply:
`STORE` info was removed.

Should we remove it from source code / Lauchpad as well (if it is not used)?